### PR TITLE
only document the code interesting for plugin creators

### PR DESCRIPTION
**Pull request by ThunderKey**

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -16,3 +16,11 @@ Code contributions
 
 Since kitovu is currently developed as an university project, we're unable to
 accept any outside code contributions. This will change around July 2018.
+
+Documenation
```
Comment (The-Compiler) [inline CONTRIBUTING.rst]: Documenation -> Documen**t**ation

File: docs/reference/smb.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Samba Plugin
```
Comment (The-Compiler) [inline docs/reference/smb.rst]: Samba -> SMB (SMB is the protocol, Samba is a Linux implementation of SMB)

**Review by The-Compiler — CHANGES_REQUESTED** [review]

File: docs/reference/smb.rst (hunk 2 above)
Comment (ThunderKey) [reply docs/reference/smb.rst]: Whoops, thanks

**Review by The-Compiler — APPROVED** [review]

**Comment by The-Compiler** [comment]
Sorry for the delay and thanks!

**Pull request merged**